packages/react-core/src/components/Tabs/TabContent.tsx

```diff
@@ -1,14 +1,22 @@
 import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/TabContent/tab-content';
 import { css } from '@patternfly/react-styles';
 import { getOUIAProps, OUIAProps } from '../../helpers';
 
+export enum TabContentVariants {
+  default = 'default',
+  light300 = 'light300'
+}
+
 export interface TabContentProps extends Omit<React.HTMLProps<HTMLElement>, 'ref'>, OUIAProps {
   /** content rendered inside the tab content area if used outside Tabs component */
   children?: any;
   /** Child to show in the content area */
   child?: React.ReactElement<any>;
   /** class of tab content area if used outside Tabs component */
   className?: string;
+  /** TabContent background color variant */
+  variant?: 'default' | 'light300';
```

Comment:
The issue with that is that light variant for other components (see PageSection) is the white background color. It would be really confusing if in one component variant="light" will result in white and in another in light grey background.

Reply:
@jschuler we aren't sure if there will be more colors, but I agree the naming could probably be improved. We tried thinking of color names that could scale, assuming we introduced "light" - would we use lighter/darker? Another problem is if we did that, and wanted to introduce a color between light and lighter - what would we call that? We also considered "inverse" (since the white/grey backgrounds are swapped in this variation), but again that doesn't scale if we offer a different color scheme, and doesn't really describe what the purpose of this color is for - just to pair with when used with content in a grey background, or more generally a page section with its default background color - `--pf-c-page__main-section--BackgroundColor: var(--pf-global--BackgroundColor--light-300);`. cc @mcarrano and @mceledonia

Not suggesting we should do this, but I think it would clear things up if both tabs and page backgrounds were white by default, and we had a common modifier name for both that made the background the grey color. Maybe it would help if we came up with a good theme name for the grey background color, something other than "light-300"?

Reply:
I see, in that case i don't have objections to the `light300` variant name. Can't think of anything better.
Also, it appears to be the scheme for the tabs and the tab content should be tightly coupled? i.e. with the light-300 scheme you'd always want the content to have that grey background as well?
In that case @KKoukiou i'd suggest we only have a top level variant prop in the Tabs component, and use context to pass that variant down to the content

Reply:
@jschuler that sounds good to me!